<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRoomDiaLog</class>
 <widget class="QDialog" name="CreateRoomDiaLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans 3 Medium</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Room</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>72</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>591</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="text">
      <string>Time Limit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="capacity_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="text">
      <string>Maximum user :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="privateRadioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator {
	width: 20px; 
	height: 20px;
}</string>
     </property>
     <property name="text">
      <string>Private</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>98</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3 SemiBold</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#nextButton {
	background-color: rgb(9, 44, 76);
	border: none;
	color: #fff;
	border-radius: 4px;
}

#nextButton:hover {
	background-color: rgba(9, 44, 76, 0.9);
}</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>260</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="capacitySpinBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>180</y>
       <width>260</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 191, 154);
border-radius: 15px;
color: #fff</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>General</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #DEE5EF;
border-radius: 15px;
color: #67758D; </string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Exam</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>24</y>
        <width>181</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #DEE5EF;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="passwordInput" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>233</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>72</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="categoryComboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>601</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Level 1:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="level1SpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Level 2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Level 4:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="level2SpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="level3SpinBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Max score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalScoreLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3 SemiBold</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>98</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3 SemiBold</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#createButton {
	background-color: rgb(9, 44, 76);
	border: none;
	color: #fff;
	border-radius: 4px;
}

#createButton:hover {
	background-color: rgba(9, 44, 76, 0.9);
}</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 191, 154);
border-radius: 15px;
color: #fff</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>General</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 191, 154);
border-radius: 15px;
color: #fff; </string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Exam</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>24</y>
        <width>181</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 191, 154);</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>98</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3 SemiBold</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#backButton {
	color: rgb(9, 44, 76);
	border: 1px solid rgb(9, 44, 76);
	border-radius: 4px;
}

#backButton:hover {
	background-color: rgba(255, 255, 255, 0.8);
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="level4SpinBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="level5SpinBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;
</string>
     </property>
     <property name="text">
      <string>Level 3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>160</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans 3</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Level 5:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
